packages/screenshots/Classes/Configuration/Configuration.php

```diff
@@ -270,6 +297,64 @@ public function createBasicConfig(): void
                             ['action' => 'click', 'link' => "List"],
                             ['action' => 'makeScreenshotOfFullPage', 'fileName' => "IntroductionDashboardFullPage"],
                         ],
+                        "startpage" => [
```

Comment:
Remove the former EXT:introduction backend screenshot section as the new backend section does the same. Split EXT:introduction backend and frontend screenshot into two sections with own identifier, as done for EXT:examples.